docs/spark-profiling-tool.md

```diff
@@ -7,7 +7,9 @@ nav_order: 9
 
 The Profiling tool analyzes both CPU or GPU generated event logs and generates information 
 which can be used for debugging and profiling Apache Spark applications.
-The output information contains the Spark version, executor details, properties, etc.
+The output information contains the Spark version, executor details, properties, etc.  
+Starting with release "_22.10_", the Profiling tool optionally provides optimized RAPIDS
```

Comment:
bold 22.10 instead of italics

Reply:
should do the same in the qualification docs? I believe the release versions are in italic.